cumulus/parachains/runtimes/collectives/collectives-westend/src/xcm_config.rs

```diff
@@ -0,0 +1,338 @@
+					pallet_collective::Call::disapprove_proposal { .. } |
+					pallet_collective::Call::close { .. },
+			) | RuntimeCall::FellowshipCollective(
+				pallet_ranked_collective::Call::add_member { .. } |
+					pallet_ranked_collective::Call::promote_member { .. } |
+					pallet_ranked_collective::Call::demote_member { .. } |
+					pallet_ranked_collective::Call::remove_member { .. },
+			) | RuntimeCall::FellowshipCore(
+				pallet_core_fellowship::Call::bump { .. } |
+					pallet_core_fellowship::Call::set_params { .. } |
+					pallet_core_fellowship::Call::set_active { .. } |
+					pallet_core_fellowship::Call::approve { .. } |
+					pallet_core_fellowship::Call::induct { .. } |
+					pallet_core_fellowship::Call::promote { .. } |
+					pallet_core_fellowship::Call::offboard { .. } |
+					pallet_core_fellowship::Call::submit_evidence { .. } |
+					pallet_core_fellowship::Call::import { .. },
+			)
+		)
+	}
+}
+
+pub type Barrier = TrailingSetTopicAsId<
+	DenyThenTry<
+		DenyReserveTransferToRelayChain,
+		(
+			// Allow local users to buy weight credit.
+			TakeWeightCredit,
+			// Expected responses are OK.
+			AllowKnownQueryResponses<PolkadotXcm>,
+			// Allow XCMs with some computed origins to pass through.
+			WithComputedOrigin<
+				(
+					// If the message is one that immediately attempts to pay for execution, then
+					// allow it.
+					AllowTopLevelPaidExecutionFrom<Everything>,
+					// Parent and its pluralities (i.e. governance bodies) get free execution.
+					AllowExplicitUnpaidExecutionFrom<ParentOrParentsPlurality>,
+					// Subscriptions for version tracking are OK.
+					AllowSubscriptionsFrom<ParentOrSiblings>,
+				),
+				UniversalLocation,
+				ConstU32<8>,
+			>,
+		),
+	>,
+>;
+
+/// Cases where a remote origin is accepted as trusted Teleporter for a given asset:
+/// - DOT with the parent Relay Chain and sibling parachains.
+pub type TrustedTeleporters = ConcreteAssetFromSystem<WndLocation>;
+
+pub struct XcmConfig;
+impl xcm_executor::Config for XcmConfig {
+	type RuntimeCall = RuntimeCall;
+	type XcmSender = XcmRouter;
+	type AssetTransactor = CurrencyTransactor;
+	type OriginConverter = XcmOriginToTransactDispatchOrigin;
+	// Collectives does not recognize a reserve location for any asset. Users must teleport WND
+	// where allowed (e.g. with the Relay Chain).
+	type IsReserve = ();
+	type IsTeleporter = TrustedTeleporters;
+	type UniversalLocation = UniversalLocation;
+	type Barrier = Barrier;
+	type Weigher = FixedWeightBounds<TempFixedXcmWeight, RuntimeCall, MaxInstructions>;
+	type Trader =
+		UsingComponents<WeightToFee, WndLocation, AccountId, Balances, ToStakingPot<Runtime>>;
+	type ResponseHandler = PolkadotXcm;
+	type AssetTrap = PolkadotXcm;
+	type AssetClaims = PolkadotXcm;
+	type SubscriptionService = PolkadotXcm;
+	type PalletInstancesInfo = AllPalletsWithSystem;
+	type MaxAssetsIntoHolding = MaxAssetsIntoHolding;
+	type AssetLocker = ();
+	type AssetExchanger = ();
+	type FeeManager = ();
```

Comment:
we've set up this for every system parachain and relay chain,
probably we should do also here
```suggestion
	type FeeManager = XcmFeeManagerFromComponents<
		WaivedLocations,
		XcmFeeToAccount<Self::AssetTransactor, AccountId, TreasuryAccount>,
	>;
```